specification/src/main/asciidoc/license-efsl.adoc

```diff
@@ -34,7 +34,7 @@ document, or portions thereof, that you use:
 * All existing copyright notices, or if one does not exist, a notice
   (hypertext is preferred, but a textual representation is permitted)
   of the form: "Copyright (c) [$date-of-document]
-  Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php"
+  Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php[]"
```

Comment:
I thought that you would need link:https://www.eclipse.org/legal/efsl.php[]

Reply:
Nope, the link: is not required.  Already verified the pdf and html generation.